public/locales/en/common.json

```diff
@@ -234,5 +234,19 @@
     "NEXT": "Next",
     "SKIP": "Skip",
     "FINISH": "Finish"
+  },
+  "BLOCKS": {
+    "TEACHING_BLOCKS": "Teaching Blocks",
+    "SEARCH" : "Search..",
+    "FILTERS": "Filters",
+    "CREATE_NEW": "Create New",
+    "NEW_BLOCK": "New Block",
+    "BLOCK_NAME": "Block Name",
+    "CREATE": "Create",
+    "NAMES": "Names",
+    "A_TO_Z": "A to Z",
+    "Z_TO_A": "Z to A",
+    "SEARCH_BLOCKS": "Search Blocks",
+    "APPLY": "Apply"
```

Comment:
@Aar-if check most of them must be there in common object or if present in other object lets move it to common object and use from there

Reply:
Okay